src/Database/Connection.php

```diff
@@ -25,15 +25,13 @@
 use Illuminate\Database\ConnectionResolverInterface;
 use Illuminate\Database\DatabaseManager;
 use Hyn\Tenancy\Events;
-use InvalidArgumentException;
 
 class Connection
 {
     use DispatchesEvents;
 
     const DEFAULT_SYSTEM_NAME = 'system';
     const DEFAULT_TENANT_NAME = 'tenant';
-    const DEFAULT_MIGRATION_NAME = 'tenant-migration';
```

Comment:
Wouldn't it be better to deprecate this const in stead of removing it? Removing it would break stuff for users.

Reply:
I should have done that indeed. Rushed decision.

Reply:
Can we still revert this in a new release?